problematic_word_extraction.py

```diff
@@ -0,0 +1,116 @@
+# Extract potentially problematic words for TTS from a CSV using a spell checker
+# Divide the words up into separate files by punctuation
+
+import pandas as pd
+# may need to do a 'pip install pyspellchecker'
+from spellchecker import SpellChecker 
+import sys
+
+# python problematic_word_extraction.py messages.csv
+
+spell = SpellChecker()
```

Comment:
Is this locale-dependent?

Reply:
ie, does it work outside of US-English?

Reply:
different languages are supported...
https://pyspellchecker.readthedocs.io/en/latest/
> The currently supported dictionaries are:

English - ‘en’
Spanish - ‘es’
French - ‘fr’
Portuguese - ‘pt’
German - ‘de’
Russian - ‘ru’

Reply:
Great, thanks for adding!